core/src/test/scala/com/evolutiongaming/catshelper/ToFutureSpec.scala

```diff
@@ -25,6 +28,8 @@ class ToFutureSpec extends AsyncFunSuite with Matchers {
 
       val future = either.toFuture
 
+      Await.ready(future, 10.seconds)
```

Comment:
same problem as with `ToTry`, we should then create custom implementation of `ToFuture`…

Reply:
Done. Please check the implementation. I think it _should_ be correct.

Reply:
@DeviLab could you please remove all added `Awaits` in this MR, as it should work without them otherwise we have bugs…

Reply:
removed